docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

```diff
@@ -81,7 +102,92 @@ test('My test', async t => {
 });
 ```
 
-## Checking if an Element is Available
+## Access Child Nodes in the DOM Hierarchy
+
+The [selector API](selectors/functional-style-selectors.md) allows you to filter matching elements and search through adjacent elements in the DOM tree.
+
+Selector API contains two types methods:
+
+* methods that enumerate all node types
+* methods that enumerate DOM elements only
+
+<!-- markdownlint-disable MD033 -->
+Methods                                                | Enumerated Nodes
+------------------------------------------------------ | -------
```

Comment:
Would it be better if we invert this table?

Enumerate All Nodes | Enumerate Element Nodes only 
------------------------------------------------------ | -------
[filter(function)](selectors/functional-style-selectors.md#filter)<br/>[find](selectors/functional-style-selectors.md#find)<br/>[parent](selectors/functional-style-selectors.md#parent) | [nth](selectors/functional-style-selectors.md#nth)<br/>[withText](selectors/functional-style-selectors.md#withtext)<br/>[withExactText](selectors/functional-style-selectors.md#withexacttext)<br/>[withAttribute](selectors/functional-style-selectors.md#withattribute)<br/>[filter(string)](selectors/functional-style-selectors.md#filter)<br/>[filterVisible](selectors/functional-style-selectors.md#filtervisible)<br/>[filterHidden](selectors/functional-style-selectors.md#filterhidden)<br/>[child](selectors/functional-style-selectors.md#child)<br/>[sibling](selectors/functional-style-selectors.md#sibling)<br/>[nextSibling](selectors/functional-style-selectors.md#nextsibling)<br/>[prevSibling](selectors/functional-style-selectors.md#prevsibling) 
<!-- markdownlint-enable MD033 -->

Reply:
I'm... not sure this is the right use of tables. That's not how people usually browse them.